test.js

```diff
@@ -1007,6 +1007,207 @@ tape('getPeers - fail pt3', t => {
   })
 })
 
+tape('register user - pass', t => {
+  const db = levelup(enc(memdown('./users.db'), { valueEncoding: 'json' }))
+
+  const registerUser = createRegisterUser(db)
+  const tx = Math.random()
+  const meta_stream = jsonStream(new PassThrough())
+  const metadata = { type: 'REGISTER', user: 'balou',
+                     peers: [], password: 'kd', tx }
+  meta_stream.once('data', res => {
+    t.true(valid.schema_RESPONSE(res), 'response is valid schema R')
+    t.true(res.ok, 'response status ok')
+    t.equal(res.tx, tx, 'transaction identifiers equal')
+    t.end()
+  })
+
+  registerUser(meta_stream, metadata, err => {
+    if(err) t.end(err)
+  })
+})
+
+tape('register user - fail pt 1 - invalid metadata', t => {
+  const db = levelup(enc(memdown('./users.db'), { valueEncoding: 'json' }))
+
+  const registerUser = createRegisterUser(db)
+  const tx = Math.random()
+  const meta_stream = jsonStream(new PassThrough())
+  const metadata = { type: 'REGISTER', user: 'balou', peers: [], tx }
```

Comment:
with the new `valid.schema_REGISTER` set `metadata.status` so it really  fails like the comments state